BGL/include/CGAL/boost/graph/copy_face_graph.h

```diff
@@ -208,7 +212,39 @@ void copy_face_graph(const SourceMesh& sm, TargetMesh& tm,
                        sm_vpm, tm_vpm);
 }
 
+
+
 } // end of namespace internal
+namespace impl
+{
+template<typename PMAP>
+struct Output_iterator_functor
+{
+  typedef typename boost::property_traits<PMAP>::key_type input_t;
+  typedef typename boost::property_traits<PMAP>::value_type output_t;
+  PMAP map;
+  Output_iterator_functor(PMAP map)
+    :map(map)
+  {
+  }
+  void operator()(const typename std::pair<input_t, output_t>& pair)
+  {
+    put(map, pair.first, pair.second);
+  }
+  
+};
+    
+template<typename PMAP>
+boost::function_output_iterator<Output_iterator_functor<PMAP> > make_functor(PMAP map)
+{
+  return boost::make_function_output_iterator(Output_iterator_functor<PMAP>(map));
+}
+
+Emptyset_iterator make_functor(const boost::param_not_found&)
```

Comment:
This global function (not templated) must be marked with `inline`. That causes all the compilation errors in demo/Polyhedron/, in all platforms:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-53.shtml#Polyhedron_Demo